Kovaleva.HomeWork-6/Card.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kovaleva.HomeWork_6
+{
+    internal class Card
+    {
+        public int CardNumber { get; set; }
+        public int Pin { get; set; }
+        public decimal Balance { get; set; }
+        public string OwnerName { get; set; }
+
+        public Card(int cardNumber, int pin, string ownerName, decimal balance = 0)
+        {
+            CardNumber = cardNumber;
+            Pin = pin;
+            OwnerName = ownerName;
+            Balance = balance;
+        }
+
+        public Card(int cardNumber, int pin)
```

Comment:
you can use:
public Card(int cardNumber, int pin) : this(cardNumber, pin, null)
or
just add ownerNumber = null to first constructor
it will improve readability and code reuse

Reply:
Можно поподробней про this? мы это еще не проходили. Вместо 2-х конструкторов лучше добавить в 1-й конструктор параметр со значением по умолчанию, правильно?

Reply:
Да, про this отписал в другом комментарии